release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -208,6 +208,14 @@ If you have already upgraded your cluster to {product-title} {product-version} a
 * An IPv6 network for API and Ingress services on the second `machineNetwork` configuration.
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.html#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network] in _Converting to IPv4/IPv6 dual-stack networking_.
+[id="ocp-4-16-networking-metallb-frr-k8s_{context}"]
+==== Integration of MetalLB and FRR-K8s (Technology Preview)
+
+This release introduces `FRR-K8s`, a Kubernetes based `DaemonSet` that exposes a subset of the `FRR` API in a Kubernetes-compliant manner.
+As a cluster administrator, you can use the `FRRConfiguration` custom resource (CR) to configure the `MetalLB` Operator to use the `FRR-K8s` daemon set as the backend.
```

Comment:
Pretty sure it's just MetalLB Operator (without backticks), unless that has changed. 

Note how DaemonSet is being used here vs. above. Maybe there's a difference in context here, in that the one above is referring to a `DaemonSet` resource and this one maybe isn't, but I'm not sure. Just thought I'd raise that scenario in the event that that is how you are using them.

```suggestion
As a cluster administrator, you can use the `FRRConfiguration` custom resource (CR) to configure the MetalLB Operator to use the `FRR-K8s` daemon set as the backend.
```